docs/release-notes/8.7.asciidoc

```diff
@@ -69,6 +69,8 @@ NOTE: This problem can occur when upgrading {agents} from 8.6.2 or 8.7.0 to any
 
 * Enabling malware protection on Linux hosts might cause {elastic-defend} to enter a failed state when mount points are unmounted, which will stop the integration from working. To fix this, turn off <<malware-protection,malware protection>> on the {elastic-defend} integration policy.
 
+* Index aliases and some data streams are not properly retrieved by the {elastic-sec} default data view.   
```

Comment:
@yctercero would you mind reviewing this for accuracy? I'll be re-using this for the fixed bug summary in 8.8.

Reply:
Confirmed with @Yara that this is indeed only the Security default data view.